packages/meso-js/README.md

```diff
@@ -63,39 +62,26 @@ used in a vanilla JavaScript application as well.
 
 ### Account setup
 
-To use the `meso-js` SDK, you must have a partner account with
-[Meso](https://meso.network). To sign up, reach out to
-[support@meso.network](mailto:support@meso.network). During the onboarding
-process, you will need to specify the
-[origin/domain](https://developer.mozilla.org/en-US/docs/Glossary/Origin) of
-your dApp or web application to ensure`meso-js` operates seamlessly on your
-site. Meso will then provide you with a `partnerId` and an npm token for SDK
-installation and configuration.
+To use the `meso-js` SDK, you must have a [Meso](https://meso.network) partner
+account. You can reach out to
+[support@meso.network](mailto:support@meso.network) to sign up. During the
+onboarding process, you will need to specify the
+[origin](https://developer.mozilla.org/en-US/docs/Glossary/Origin) of your dApp
+or web application to ensure`meso-js` operates seamlessly on your site. Meso
```

Comment:
Nit: missing space between "ensure" and "`meso-js`"

Reply:
Good catch. Thanks!